databag/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+
+  app_proxy:
+    environment:
+      APP_HOST: databag_web_1
+      APP_PORT: 7777
+      
+  web:
+    image: balzack/databag:latest
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
```

Comment:
```suggestion
```
ports are not explicitly required as we are using `app_proxy` which handles the port mapping.

Reply:
@IMPranshu suggestion applied, thank you!

Reply:
Did you try running the app once after applying the changes? Do try once. Will help in opening the next app submission PR.

Reply:
I am traveling for the next week; I will try when I return.